src/main/java/net/pms/configuration/MapFileConfiguration.java

```diff
@@ -66,60 +67,65 @@ public MapFileConfiguration() {
 			conf = null;
 
 			try {
-				conf = FileUtils.readFileToString(file);
-			} catch (FileNotFoundException ex) {
-				LOGGER.warn("Can't read file: {}", ex.getMessage());
-				return null;
+				conf = FileUtils.readFileToString(file, StandardCharsets.US_ASCII);
```

Comment:
@valib is there a reason why you used US_ASCII instead of UTF8?

Reply:
@SubJunk I tried to use something general. The question is what is the general charset for all .confs? If I change it in my WIN10 the charset used by Notepad or other text editor is Windows CP1250.

Reply:
I think UTF-8 is a de facto standard that supports a lot of different characters but maybe my information is out of date. It tends to be the encoding type that databases are initialised to unless there is a good reason otherwise

Reply:
UTF-8 is the best choice when you choose a text file encoding for something new. However, it doesn't help to read something as UTF-8 if the source isn't encoded as UTF-8. The first 127 codes of UTF-8 are the same as US-ASCII, but that's also true for many others, like for example ISO 8859-1.

The Java "properties" files are defined as using ISO 8859-1, which is why unicode characters must be specified using `\uxxxx`. Since the `.conf` files used by UMS are properties files, they all have to be encoded as ISO 8859-1. It might be confusing for a user to be expected to use ISO 8859-1 in most configuration files, but UTF-8 in the virtual folders configuration file. At the same time, if this is a JSON file, the `\uxxxx` notation probably isn't supported so that would make it impossible to use unicode characters in virtual folders.